### PR TITLE
docs: add stage0-finishers.md

### PR DESCRIPTION
**Pull request by Shramkoweb**

### CODE REVIEW

**Review by miroslav-kolomiets — APPROVED** [review]

**Comment by OleksiyRudenko** [comment]
@Shramkoweb thank you for the proposal.
I will suggest a few amendments later to exclude badge-related parts as we do not assign badges this time.

File: students/stage0-finishers.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ * **Course repo** is your nick name with a link to your
+   course repo (`kottans-frontend`)
+ * **R** means **Regulars** and is a number of regular tasks completed;
+   normally says `6`; if `5` or lesser then you can
+   be inquired for the reasons
+ * **P** means **Practicals** and is a number of practical tasks completed;
+   normally says `7`; if `6` or lesser then give a reason in **Comment**
+ * **O** means **Optionals** and is a number of optional tasks
+   completed;
+   `0` is OK; `2` qualifies you for a special badge
```
Comment (OleksiyRudenko) [inline students/stage0-finishers.md:37]:
```suggestion
   `0` is OK; `2` is good for a student as optional tasks offer more knowledge
```

File: students/stage0-finishers.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+   provided all mandatory tasks are completed
```
Comment (OleksiyRudenko) [inline students/stage0-finishers.md:38]:
```suggestion
```

File: students/stage0-finishers.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+ * **B** means **Badges** and is a number of badges earned during
```
Comment (OleksiyRudenko) [inline students/stage0-finishers.md:39]:
```suggestion
```

File: students/stage0-finishers.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+   the course Stage 0; `0` is OK; anything above that is a plus
```
Comment (OleksiyRudenko) [inline students/stage0-finishers.md:40]:
```suggestion
```

File: students/stage0-finishers.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+ * **Comment** is for optional comments and reasons to skip any mandatory
```
Comment (OleksiyRudenko) [inline students/stage0-finishers.md:41]:
```suggestion
 * **Student's Comment** is for optional comments and reasons to skip any mandatory
```

File: students/stage0-finishers.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+   tasks
+ * **Performance** is intended for mentors team use, e.g.
```
Comment (OleksiyRudenko) [inline students/stage0-finishers.md:43]:
```suggestion
 * **Mentors' comment** is intended for mentors team use
```

File: students/stage0-finishers.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+   to specify performance badges granted
```
Comment (OleksiyRudenko) [inline students/stage0-finishers.md:44]:
```suggestion
```

File: students/stage0-finishers.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+
+Use the first table row as a template.
+
+## Finishers
+
+|  |    Date    | Course repo    | R | P | O | B | Comment | Performance |
```
Comment (OleksiyRudenko) [inline students/stage0-finishers.md:50]:
```suggestion
|  |    Date    | Course repo    | R | P | O | Student's Comment | Mentors' Comment |
```

File: students/stage0-finishers.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+|--| ---------- | -------------- | - | - | - | - | ------- | ----------- |
```
Comment (OleksiyRudenko) [inline students/stage0-finishers.md:51]:
```suggestion
|--| ---------- | -------------- | - | - | - | ------- | ----------- |
```

File: students/stage0-finishers.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+|1 | 2020-10-27 | [Shramkoweb](https://github.com/Shramkoweb/kottans-frontend) | 6 | 7 | 0 | 0 | --- | |
```
Comment (OleksiyRudenko) [inline students/stage0-finishers.md:52]:
```suggestion
|1 | 2020-10-27 | [Shramkoweb](https://github.com/Shramkoweb/kottans-frontend) | 6 | 7 | 0 | --- | |
```

**Review by OleksiyRudenko — CHANGES_REQUESTED** [review]
Thank you for the submission. Please incorporate suggestions below

**Comment by OleksiyRudenko** [comment]
@miroslav-kolomiets please review the changes

**Review by AMashoshyna — APPROVED** [review]

**Review by Croftyland — APPROVED** [review]

**Review by OleksiyRudenko — APPROVED** [review]

**Pull request merged**